tests/integration/data/kfp_1.7_stable_generic_install.yaml.j2

```diff
@@ -0,0 +1,198 @@
+bundle: kubernetes
+applications:
+  argo-controller:
+    charm: argo-controller
+    channel: 3.3/stable
+    revision: 236
```

Comment:
Are the revisions here intentional?  The file name says "1.7_stable", but with the revisions here this wont actually keep up with 1.7's stable but instead be whatever 1.7 stable was when the file was created.  

I wonder if this is an export from the `juju export-bundle` command, and some attributes need to be removed before including it here?

Reply:
It is an export, that's correct, I did not bother removing the revisions and other attributes, but I can do it. Good catch.

Reply:
Updated.